pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -307,6 +308,14 @@ def make_package(args):
             else:
                 shutil.copytree(realpath(asset_src), join(assets_dir, asset_dest))
 
+        for res in args.resources:
+            res_src, res_dest = res.split(":")
+            if isfile(realpath(res_src)):
+                ensure_dir(dirname(join(res_dir, res_dest)))
```

Comment:
This is obviously redundant with the assets copying logic just above, and could be factorized if needed